### PR TITLE
Fix rule-based action meta-attributes

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , can you review this when you get a chance? Thanks

**Review by nrwahl2 — COMMENTED** [review]
Great

File: lib/pengine/pe_actions.c [hunk 1]
```diff
@@ -668,135 +664,191 @@ find_min_interval_mon(pe_resource_t * rsc, gboolean include_disabled)
...
-     *      by default for a recurring monitor)
-     *   2. explicit op timeout on the primitive
-     *   3. default op timeout
-     *      a. if probe, then min-interval monitor's timeout
-     *      b. else, in XML_CIB_TAG_OPCONFIG
+     * Timeout order of precedence (highest to lowest):
+     *   1. pcmk_monitor_timeout (for starts and probes, if rsc has
+     *      pcmk_ra_cap_fence_params; for recurring monitors, it is used via the
+     *      executor instead)
+     *   2. Explicit timeout in op
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pengine/pe_actions.c:777]: might as well

File: lib/pengine/pe_actions.c [hunk 2]
```diff
@@ -668,135 +664,191 @@ find_min_interval_mon(pe_resource_t * rsc, gboolean include_disabled)
...
-                                           action->rsc->cluster);
+        && (pcmk__str_eq(action_name, PCMK_ACTION_START, pcmk__str_none)
+            || pcmk_is_probe(action_name, interval_ms))) {
+
+        GHashTable *params = pe_rsc_params(rsc, node, rsc->cluster);
+
+        timeout_spec = g_hash_table_lookup(params, "pcmk_monitor_timeout");
+        if (timeout_spec != NULL) {
+            pe_rsc_trace(rsc,
+                         "Setting default timeout for %s %s to "
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pengine/pe_actions.c:793]: good point

**Comment by kgaillot** [comment]
updated commits per review

**Pull request merged**